src/elements/content-explorer/ItemGridCell.js

```diff
@@ -24,10 +26,20 @@ const ItemGridCell = ({
     view,
     ...rest
 }: Props) => {
+    const url = getProp(item, `thumbnailUrl`);
+    const itemClassName = classNames(
+        'bce-ItemGridCell-item',
+        { 'bce-ItemGridCell-item--postLoadThumbnail': url },
+        { 'bce-ItemGridCell-item--itemIcon': !url },
+    );
     return (
-        <figure>
+        <figure className="bce-itemGridCell-figure">
```

Comment:
`figure` was adding 40px margins on the right and left sides of the content.  This made the thumbnails noticeably smaller.

Reply:
Is that from user-agent styles? We might want to add that _reset.scss

Reply:
Yes it is.  I'll make that change in `_reset.scss`.  This appears to be the only place we use `figure` in elements, so the change shouldn't affect anything else.

Reply:
This is also the root element in `ItemGridCell`, so it's class name should be `.bce-ItemGridCell`.